### PR TITLE
Jcaccamo/readme phase2 cpp cloud

### PR DESCRIPTION
**Pull request by jared-esri**
@ldanzinger Ready for review. Had to get creative with a few but seems good to me.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/IntegratedWindowsAuthentication/README.md [hunk 1]
```diff
@@ -20,23 +20,29 @@ IWA, which is built into Microsoft Internet Information Server (IIS), works well
...
-3. When a search for portal items is performed against an IWA-secured portal, the challenge handler creates an `UserCredential` object from the information entered by the user or what was automatically obtained by the current Windows login.
-4. If the user authenticates, the search returns a list of web maps from `PortalItem` objects and the user can select one to display as an `Map`.
+1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username, password, and domain) if a secure resource is encountered.
+2. When a search for portal items is performed against an IWA-secured portal, the challenge handler creates an `ArcGISNetworkCredential` object from the information entered by the user.
+3. If the user authenticates, the search returns a list of web maps (`ArcGISPortalItem`) and the user can select one to display as a `Map`.
+4. On some platforms, the current Windows account is used by default and a login prompt will not be shown if it can authenticate successfully.
 
 ## Relevant API
 
+* ArcGISNetworkCredential
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/IntegratedWindowsAuthentication/README.md]: In Qt it is just called Credential

File: ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/IntegratedWindowsAuthentication/README.md [hunk 2]
```diff
@@ -20,23 +20,29 @@ IWA, which is built into Microsoft Internet Information Server (IIS), works well
...
+* ArcGISPortal
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/IntegratedWindowsAuthentication/README.md]: Should just be Portal

File: ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/IntegratedWindowsAuthentication/README.md (hunk 1 above)
Comment (ldanzinger) [reply ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/IntegratedWindowsAuthentication/README.md]: (above as well in how it works)

File: ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/IntegratedWindowsAuthentication/README.md (hunk 2 above)
Comment (ldanzinger) [reply ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/IntegratedWindowsAuthentication/README.md]: (above as well in how it works)

**Comment by jared-esri** [comment]
Should be good to go.

**Pull request merged**